File: 2nd year/Java/Samp.java
import java.util.Scanner;
import java.util.Random;

public class Samp {
    public static void main(String[] args) {
        Random rand = new Random();
        int numberToGuess = rand.nextInt(100);
        int numberOfTries = 0;
        Scanner input = new Scanner(System.in);
        int guess;
        boolean win = false;

        while (win == false) {
            System.out.println("Guess a number between 1 and 100: ");
            guess = input.nextInt();
            numberOfTries++;

            if (guess == numberToGuess) {1
                win = true;
            } else if (guess < numberToGuess) {
                System.out.println("Your guess is too low");
            } else if (guess > numberToGuess) {
                System.out.println("Your guess is too high");
            }
        }

        System.out.println("You win!");
        System.out.println("The number was " + numberToGuess);
        System.out.println("It took you " + numberOfTries + " tries");
    }
}

File: 2nd year/Python/Samp.py
import random

# Generate a random number between 1 and 1000
number = random.randint(1, 1000)

# Set the number of guesses to 0
guesses = 0

# Loop until the user guesses the correct number
while True:
    # Ask the user to guess a number
    guess = int(input("Guess a number between 1 and 1000: "))

    # Increment the number of guesses
    guesses += 1

    # Check if the guess is correct
    if guess == number:
        print("Congratulations! You guessed the number in", guesses, "guesses.")
        break
    elif guess < number:
        print("Too low. Guess again.")
    else:
        print("Too high. Guess again.")100